distribution/package_publish_workspace.sh

```
@@ -8,7 +8,7 @@ cd ..
 
 echo "Starting to package Rust workspace projects..."
 
-PROJECTS=("rocketmq-common" "rocketmq-runtime" "rocketmq-macros" "rocketmq" "rocketmq-filter" "rocketmq-store" "rocketmq-remoting" "rocketmq-cli" "rocketmq-example" "rocketmq-client" "rocketmq-namesrv" "rocketmq-broker")
+PROJECTS=("rocketmq-common" "rocketmq-runtime" "rocketmq-macros" "rocketmq" "rocketmq-filter" "rocketmq-store" "rocketmq-remoting" "rocketmq-cli" "rocketmq-example" "rocketmq-client" "rocketmq-namesrv" "rocketmq-broker","rocketmq-tools")
```

Shellcheck (0.10.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)